src/main/java/appeng/fluids/parts/PartSharedFluidBus.java

```diff
@@ -149,7 +155,7 @@ public IItemHandler getInventoryByName( final String name )
 	{
 		if( name.equals( "config" ) )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "config" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "config" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1132)